draft-packages/avatar/docs/AvatarGroup.stories.tsx

```diff
@@ -114,3 +115,21 @@ StickerSheetReversed.parameters = {
   backgrounds: { default: "Purple 700" },
   chromatic: { disable: false },
 }
+
+const StickerSheetTemplateComponent = StickerSheetTemplate.bind({})
+StickerSheetTemplateComponent.parameters = { chromatic: { disable: false } }
+
+export const StickerSheetRtl = () => (
+  <div dir="rtl">
+    <StickerSheetTemplateComponent isReversed={false} />
+  </div>
+)
+StickerSheetRtl.storyName = "Sticker Sheet (RTL)"
+
+const twoAvatars: AvatarList = [EXAMPLE_USER_1, EXAMPLE_USER_2]
+export const StickerSheetRtlWithTwoAvatars = () => (
+  <div dir="rtl">
+    <StickerSheetTemplateComponent isReversed={false} avatars={twoAvatars} />
+  </div>
+)
+StickerSheetRtlWithTwoAvatars.storyName = "Sticker Sheet (Two avatars, RTL)"
```

Comment:
Rather than adding extra sticker sheets, I'd suggest adding extra rows into StickerSheetTemplate instead. Visual regression testing can get costly as each snapshot adds cost, so we specifically try and keep them in a single story :)